fskeystore.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"path/filepath"
 
-	ci "github.com/libp2p/go-libp2p-crypto"
```

Comment:
gx paths in small deps are not needed, we use it mostly in go-ipfs to enforce the versions.